src/kolmogorov_smirnov.jl

```diff
@@ -194,11 +191,30 @@ end
 # compute supremum of differences between empirical cdfs.
 function ksstats(x::AbstractVector{T}, y::AbstractVector{S}) where {T<:Real, S<:Real}
     n_x, n_y = length(x), length(y)
-    sort_idx = sortperm([x; y])
-    pdf_diffs = [ones(n_x)/n_x; -ones(n_y)/n_y][sort_idx]
-    cdf_diffs = cumsum(pdf_diffs)
-    δp = maximum(cdf_diffs)
-    δn = -minimum(cdf_diffs)
-    δ = max(δp, δn)
-    (n_x, n_y, δ, δp, δn)
-end
+    all_values = [x; y]
+    sort_idx = sortperm(all_values)
+    δ_y = 1. / n_y
+    δ_x = 1. / n_x
+    δ = 0.
+    δp = 0.
+    δn = 0.
```

Comment:
```suggestion
    δ_y = 1 / n_y
    δ_x = 1 / n_x
    δ = δp = δn = zero(δ_y)
```
It's better to avoid using `Float64` literals whenever they aren't actually necessary to avoid unnecessary promotion. Generally `length` will return an `Int64` and `inv(::Int64)` returns `Float64` so in this particular case it's more of an academic concern than a practical one but might as well use good style.